### ISSUE TITLE
Ability to exclude shows that have not ended

### ISSUE THREAD

**Issue by nylonee** [rfsbraz/deleterr#58]
This mainly applies to TV Shows, but I imagine there might be a world in anime/movies where the same configuration might be useful.

I want Deleterr to not delete shows that are continuing, only shows that have ended. This is useful if I am waiting around for a new season of a show.

I think Sonarr has information on a show that includes whether it's continuing/ended, so this might be a simple check against Sonarr?

**Comment by rfsbraz** [comment]
Yeah this is not hard to add, I'll work on supporting show's status!

**Comment by nylonee** [comment]
I just did a similar implementation in [my own project](https://github.com/nylonee/watchlistarr), it wasn't hard at all!
When fetching the list of shows from Sonarr, there are two possible fields you can use:  
`status` (which is either `"continuing"` or `"ended"`)
`ended` (true or false)

However in some rare cases, neither of those fields show up, I haven't quite tracked down what edge cases cause that yet.

**Comment by Highlifez** [comment]
Has this ever been implemented? This would be great to have, as I don't want to delete/unmonitor ongoing shows that I'm interested in

**Comment by rfsbraz** [comment]
Hey! I have this nearly ready, I'll link it here once I have something up and running
